course/tests/courselib_test.php

```diff
@@ -575,10 +575,6 @@ public function test_create_course() {
         $sectioncreated = $DB->record_exists('course_sections', array('course' => $created->id, 'section' => 0));
         $this->assertTrue($sectioncreated);
 
-        // Ensure blocks have been associated to the course.
-        $blockcount = $DB->count_records('block_instances', array('parentcontextid' => $context->id));
-        $this->assertGreaterThan(0, $blockcount);
```

Comment:
Looking at [core 3.1](https://github.com/moodle/moodle/blob/fae923693963337cc26a2406c04c683472e15d03/course/tests/courselib_test.php#L578-L580) it seems this code is still present and I can't see it being removed in [the original diff](https://github.com/moodle/moodle/compare/7eb34671c13b07...snake:MDL-54751-master-v5#diff-d03969f0828f2cdbe16265af32f99c64). It would make sense that it has been removed in future Moodle versions where blocks are largely irrelevant in courses.

Reply:
:heavy_check_mark: Re-added